patches/chrome-browser-resources-settings-settings_shared.css.patch

```diff
@@ -0,0 +1,16 @@
+diff --git a/chrome/browser/resources/settings/settings_shared.css b/chrome/browser/resources/settings/settings_shared.css
+index b28284ade36d1ad8a695e6532b74edc406dcc067..0551ec31a00112d59d80c1265e8c100bbba75407 100644
+--- a/chrome/browser/resources/settings/settings_shared.css
++++ b/chrome/browser/resources/settings/settings_shared.css
+@@ -7,9 +7,10 @@
+  * #import=//resources/cr_elements/cr_shared_style.css.js
+  * #import=//resources/cr_elements/cr_shared_vars.css.js
+  * #import=//resources/cr_elements/search_highlight_style.css.js
++ * #import=//resources/page_specific/settings/br_settings_shared.css.js
+  * #import=./settings_vars.css.js
+ <if expr='not chromeos_ash'>
+- * #include=cr-shared-style search-highlight-style
++ * #include=cr-shared-style search-highlight-style br-settings-shared
```

Comment:
I believe our lines can be close to each other:
```
<if expr='not chromeos_ash'>
 * #import=//resources/page_specific/settings/br_settings_shared.css.js
 * #include=cr-shared-style search-highlight-style br-settings-shared
```

Reply:
Moved.